policy/resolver.go

```diff
@@ -586,6 +591,29 @@ func NewPolicyAssetMatchError(assetFilters []*explorer.Mquery, p *Policy) error
 	return status.Error(codes.InvalidArgument, msg)
 }
 
+func NewPolicyNoQueriesError(executionJob *ExecutionJob, p *Policy) error {
+	if len(executionJob.Queries) == 0 {
+		// send a proto error with details, so that the agent can render it properly
+		msg := "resolved policy does not contain any queries"
+		st := status.New(codes.NotFound, msg)
```

Comment:
When we could use the `explorer.NoQueries` code here, that would make other parts of the code easier.

I'm not sure whether I missed something here or it just isn't possible. I didn't manage to replace the code.